src/storage/src/hummock/compactor/compaction_utils.rs

```diff
@@ -199,9 +200,14 @@ pub async fn generate_splits(
         indexes.sort_by(|a, b| KeyComparator::compare_encoded_full_key(a.1.as_ref(), b.1.as_ref()));
         let mut splits = vec![];
         splits.push(KeyRange_vec::new(vec![], vec![]));
+        let cpu_core = thread::available_parallelism()?.get();
```

Comment:
Incorrect, not all `physical cores` are used in all cases, this should rely on `worker_num`